scripts/remove-redundant-flags.js

```diff
@@ -0,0 +1,187 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import fs from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import compareVersions from 'compare-versions';
+import esMain from 'es-main';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+
+import bcd from '../index.js';
+import { IS_WINDOWS } from '../test/utils.js';
+
+const dirname = fileURLToPath(new URL('.', import.meta.url));
+
+const getEarliestVersion = (...args) => {
+  const versions = args
+    .filter((version) => typeof version === 'string')
```

Comment:
This should make the conditions in the loop easier to reason about:

```suggestion
    .filter((version) => typeof version === 'string' && version !== 'preview')
```